### PR TITLE
[QMR] Build cleanup

### PR DESCRIPTION
**Pull request by tblackwe**
## Description

Refactoring Github Actions to improve developer experience.
Unit tests now enforced prior to local commit

Unit tests and Cypress tests removed from deploy stage and added to pull-request

### How to test

Review the build logs.

### Changed Dependencies

N/A

## Code author checklist

- [X] I have performed a self-review of my code
- [X] I have added [thorough](https://bit.ly/3zPrxuZ) tests
- [ ] I have added analytics, if necessary
- [ ] I have updated the documentation, if necessary

## Reviewer checklist (two different people)

- [ ] I have done the deep review and verified the items in the above checklist are g2g
- [x] I have done the lgtm review

### CODE REVIEW

File: tests/cypress/cypress/integration/a11y/LBWCHpage.spec.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-describe("LBW-CH Page 508 Compliance Test", () => {
+describe.skip("LBW-CH Page 508 Compliance Test", () => {
```
Comment (tblackwe) [inline tests/cypress/cypress/integration/a11y/LBWCHpage.spec.ts:1]: Skipping, not in original deploy script

File: tests/cypress/cypress/integration/a11y/PDENTpage.spec.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-describe("PDENT-CH Page 508 Compliance Test", () => {
+describe.skip("PDENT-CH Page 508 Compliance Test", () => {
```
Comment (tblackwe) [inline tests/cypress/cypress/integration/a11y/PDENTpage.spec.ts:1]: Skipping, not in original deploy script

File: tests/cypress/cypress/integration/a11y/nciddsPage.spec.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
-describe("NCIDDS-AD Page 508 Compliance Test", () => {
+describe.skip("NCIDDS-AD Page 508 Compliance Test", () => {
```
Comment (tblackwe) [inline tests/cypress/cypress/integration/a11y/nciddsPage.spec.ts:1]: Skipping, not in original deploy script

File: tests/cypress/cypress/integration/features/export_all_measures.spec.ts [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import { measureAbbrList2021 } from "../../../support/commands/commands";
 
-describe("Export All Measures", () => {
+describe.skip("Export All Measures", () => {
```
Comment (tblackwe) [inline tests/cypress/cypress/integration/features/export_all_measures.spec.ts:3]: Skipping, not in original deploy script

File: tests/cypress/cypress/integration/features/kebab_menu_measures.spec.ts [hunk 5]
```diff
@@ -1,4 +1,4 @@
-describe("Measure kebab menus", () => {
+describe.skip("Measure kebab menus", () => {
```
Comment (tblackwe) [inline tests/cypress/cypress/integration/features/kebab_menu_measures.spec.ts:1]: Skipping, not in original deploy script

File: tests/cypress/cypress/integration/features/kebab_menu_measures.spec.ts [hunk 6]
```diff
@@ -26,7 +26,7 @@ describe("Measure kebab menus", () => {
   });
 });
 
-describe.only("Edit state specific measure name and description", () => {
+describe.skip("Edit state specific measure name and description", () => {
```
Comment (tblackwe) [inline tests/cypress/cypress/integration/features/kebab_menu_measures.spec.ts:29]: Skipping, not in original deploy script

File: tests/cypress/cypress/integration/features/partially_completed_validation.spec.ts [hunk 7]
```diff
@@ -1,4 +1,4 @@
-describe("OY2 16341 NDR set validation updates for all measures ", () => {
+describe.skip("OY2 16341 NDR set validation updates for all measures ", () => {
```
Comment (tblackwe) [inline tests/cypress/cypress/integration/features/partially_completed_validation.spec.ts:1]: Skipping, not in original deploy script

File: tests/cypress/cypress/integration/features/state_specific_measures.spec.ts [hunk 8]
```diff
@@ -1,4 +1,4 @@
-describe("Add state specific measure testing", () => {
+describe.skip("Add state specific measure testing", () => {
```
Comment (tblackwe) [inline tests/cypress/cypress/integration/features/state_specific_measures.spec.ts:1]: Skipping, not in original deploy script

File: tests/cypress/cypress/integration/features/submit_coreset.spec.ts [hunk 9]
```diff
@@ -1,4 +1,4 @@
-describe("Submit Core Set button", () => {
+describe.skip("Submit Core Set button", () => {
```
Comment (tblackwe) [inline tests/cypress/cypress/integration/features/submit_coreset.spec.ts:1]: Skipping, not in original deploy script

**Review by BearHanded — APPROVED** [review]

**Review by ntsummers1 — APPROVED** [review]

**Pull request merged**